### PR TITLE
Added Selenoid

### PR DESCRIPTION
**Pull request by vania-pooh**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

- [X] Your additions are ordered alphabetically.
- [X] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [X] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [X] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant issues or PRs.
- [X] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1042,6 +1042,7 @@ See **[Ticketing](#ticketing)**
   * [Microglark](http://micro.glark.io) - A hacky minimalistic remote pair programming editor. ([Demo](http://micro.glark.io), [Source Code](https://github.com/Bluefinch/microglark)) `AGPLv3` `Nodejs`
   * [Regexr](http://regexr.com/) - RegExr is a HTML/JS based tool for creating, testing, and learning about Regular Expressions. ([Source Code](https://github.com/gskinner/regexr/)) `MIT` `Nodejs`
   * [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) - Web browser based IDE for R. ([Source Code](https://github.com/rstudio/rstudio/)) `AGPLv3` `Java/C++`
+  * [Selenoid](http://aerokube.com/selenoid/latest/) - A lightweight Selenium hub implementation launching browsers within Docker containers. ([Source Code](https://github.com/aerokube/selenoid)) `Apache` `Go`
```
Comment (Kickball) [inline README.md]: @nodiscc is that the correct way of referencing the Apache version 2 license?

Comment (nodiscc) [inline README.md]: https://github.com/Kickball/awesome-selfhosted#list-of-licenses yes

Comment (nodiscc) [inline README.md]: @vania-pooh Please remove the source code link as it is the same as the main link.

**Comment by vania-pooh** [comment]
Thanks.

**Pull request merged**